# Copyright 2020 RUVU Robotics B.V.

string remote_device_id
time stamp
float64 distance # meter
int16 RSS # dBm
